app/src/main/java/com/jjswigut/matters/database/DummyMatter.kt

```diff
@@ -0,0 +1,5 @@
+package com.jjswigut.matters.database
+
+data class DummyMatter(
+    var matterTitle: String, var matterContent: String
```

Comment:
in general my rule of thumb is to make everything a val until it needs to be a var